src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,5 @@
+url=jdbc:h2:mem:book_store
```

Comment:
why Liquibase works with H2, while Spring Boot works with MySQL?

Reply:
I had a problem when I use MySQL here, the project can't be built in GitHub. But locally everything is okay, and I use MySQL